.pylintrc

```diff
@@ -14,7 +14,9 @@ disable=
     too-many-arguments,
     too-many-branches,
     too-many-boolean-expressions,
+    too-many-locals,
```

Comment:
Similar to our conversations about type annotations/mypy... how often does pylint catch actual issues?

Reply:
the main benefit is catching unused variables, other than that simple formatting stuff